src/views/BridgeTransaction.vue

```diff
@@ -377,6 +420,8 @@ export default class BridgeTransaction extends Mixins(TranslationMixin, LoadingM
 
   async created (): Promise<void> {
     if (this.isTransactionConfirmed) {
+      await this.getNetworkFee()
```

Comment:
It doesn't really matter now, but we can send this fee requests parallel using `Promise.all `